uiowa/uiowa
# Hotfix: Trusted callback updates

Resolves issues related to `TrustedCallbackInterface` that were causing WSOD on https://sitenow.uiowa.edu/request-site

# How to test

* `blt ds --site=sitenow.uiowa.edu`
* Visit http://sitenow.local.drupal.uiowa.edu/request-site to see that it is not broken.
